gecko/glue/GeckoMediaDecoderOwner.cpp

```diff
@@ -255,4 +255,40 @@ GeckoMediaDecoderOwner::UpdateCurrentImages(nsTArray<GeckoPlanarYCbCrImage> aIma
   }
 }
 
+void
+GeckoMediaDecoderOwner::NotifyBuffered() const
+{
+  if (mCallback.mContext && mCallback.mNotifyBuffered) {
```

Comment:
Mozilla convention would be to exit early on a guard statement, rather than carry an indent for a non-trivia amount of code, i.e.:
``
   if (!mCallback.mContext || !mCallback.mNotifyBuffered) {   
     return;   
   }   
``

The motivation behind that is that it's less mental overhead when reading code.